src/library/module.cpp

```diff
@@ -82,6 +86,9 @@ extern "C" object * lean_read_module_data(object * fname, object * w) {
         in.close();
         /* We don't free compacted_region objects */
         compacted_region * region = new compacted_region(size - header_size, buffer);
+#if defined(__has_feature) && __has_feature(address_sanitizer)
+        __lsan_ignore_object(region);
+#endif
         object * mod = region->read();
```

Comment:
What is this?

Reply:
Added a comment. I'd like to add sanitized builds to our CI in the future so that we immediately become aware of such problems.